bookmarks.html

```diff
@@ -29,7 +30,7 @@ <h2 class="card__h2">Question:</h2>
           ipsum dolor sit amet.
         </p>
         <details>
-          <summary>Show Answers</summary>
+          <summary data-js="card__show-answer-button">Show Answers</summary>
           <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</p>
```

Comment:
for semantic html i believe, that you don t need a `p`Tag here, `details`is the container for the text shown by click the summary tag